src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form data-qa="big">
+      <div class="search-bar search-bar--larger" tabindex="1">
```

Comment:
it is redundant to make search-bar--larger modifier for big form. set all styles in search-bar and change necessary in search-bar-mini modifier for small form.
same for other elements.

Reply:
but in this case the CSS file will become larger because many identical lines will appear

Reply:
Mant thx for comments